docs/user-guide/cli-db2-install-m1.md

```diff
@@ -31,8 +31,12 @@ Use the following steps to configure an M1 (or later architecture) processor to
 5. Reinstall Zowe CLI.
 6. After you complete these steps, do one of the following:
 
-    - If you are installing the plug-in from an online registry, continue with Step 2 in [Install from an online registry](.//cli-db2plugin.md#installing-from-an-online-registry).
+    - If you are installing the plug-in from an online registry, continue with Step 2 in [Installing from an online registry](.//cli-db2plugin.md#installing-from-an-online-registry).
 
     - If you are installing the plug-in from a local package, continue with Step 2 in [Installing from a local package](../user-guide/cli-db2plugin.md#installing-from-a-local-package). 
 
-**Important!** You must issue the `intel` command **every time** that you open a new terminal window to help ensure that Zowe CLI, Secure Credential Storage and the DB2 plug-in function properly on x86_64 architecture. Also, issue the command **before** you issue Zowe CLI commands.
\ No newline at end of file
+:::warning Important 
+
+You must issue the `intel` command every time that you open a new terminal window to help ensure that Zowe CLI, secure credential storage **[is it correct to include scs here?]**, and the DB2 plug-in function properly on x86_64 architecture. Also, issue the command *before* you issue Zowe CLI commands.
```

Comment:
@adam-wolfe - please respond

Reply:
I'm not sure, I think the main concern here is the DB2 plug-in. I think it would be clearer to change this to "...window to ensure that the DB2 plug-in functions properly on x86_64 architecture."